packages/select/src/lib/select/index.tsx

```diff
@@ -194,9 +194,11 @@ function NativeMenu(
   );
 }
 
-const AnchorEl = withRipple({ surface: false })(function (props: any) {
-  return <Tag {...props} />;
-});
+const AnchorEl = withRipple({ surface: false })(
```

Comment:
I fixed this yesterday a bit differently: https://github.com/rmwc/rmwc/pull/1372/files#diff-f37a37b3d7d9a35bde510dec5925e91640816087541b38928e6604298e2e9fff

But seeing your fix here, I wonder which is better.

Reply:
After your comment, I wondered as well, so I dug a bit deeper. `withRipple` is a HOC that only does something, when the `ripple` prop is passed.

Since the `AnchorEl` is only used once and the prop is not passed, I guess it is safe to remove the call altogether.
I will therefore close this PR as you already fixed it.